### ISSUE TITLE
Docs's CSS files should be handled in docs

### ISSUE THREAD

**Issue by ffoodd** [ffoodd/a11y.css#450]
Isolated levels CSS files —e.g. `a11y-en_errors-only.css`—  only used in the docs site should be part of the site assets, not in `src.` nor `dist`.
